src/r5f1026a/platform.c

```diff
@@ -447,14 +481,16 @@ bool platform_poke(uint8_t addr, uint8_t reg, uint16_t val) {
 
   // TX requested so disable RXEN
   if (reg == 0x30 && (val & (1 << 6))) {
-    P1_bit.no0 = 1; // P10 (RXEN) is high
+    //P1_bit.no0 = 1; // P10 (RXEN) is high
+    return false; // HW PTT override
```

Comment:
I think https://github.com/OpenRTX/sa8x8-fw/pull/17#issuecomment-2068513388 can be addressed by simply removing the `return false` statements here and on  line 493.

Reply:
I would recommend making it an "OR" logic.  If either the serial command, OR the hardware pin are active, then put the AT1846 into transmit mode.  If neither are active, then put the AT1846 into receive mode.